app/indexers/app_indexer.rb

```diff
@@ -23,21 +23,21 @@ def generate_solr_document
     end
   end
 
-  def all_decendent_file_sets(o)
+  def all_decendent_file_sets(object)
```

Comment:
April TODO: Put this back from object to o -- I needed it visually to make sense.